cmd/clusterctl/cmd/config_repositories_test.go

```diff
@@ -137,6 +140,7 @@ vcd                 InfrastructureProvider   https://github.com/vmware/cluster-a
 vcluster            InfrastructureProvider   https://github.com/loft-sh/cluster-api-provider-vcluster/releases/latest/                   infrastructure-components.yaml
 virtink             InfrastructureProvider   https://github.com/smartxworks/cluster-api-provider-virtink/releases/latest/                infrastructure-components.yaml
 vsphere             InfrastructureProvider   https://github.com/kubernetes-sigs/cluster-api-provider-vsphere/releases/latest/            infrastructure-components.yaml
+helm                AddonProvider            https://github.com/kubernetes-sigs/cluster-api-addon-provider-helm/releases/latest/         add-on-components.yaml
```

Comment:
```suggestion
helm                AddonProvider            https://github.com/kubernetes-sigs/cluster-api-addon-provider-helm/releases/latest/         addon-components.yaml
```

Looks like the test is failing because of this missing fix.